api/data/sections/sections-json-parser.ts

```diff
@@ -0,0 +1,32 @@
+import { Section } from '@shared/dto/sections/section.entity';
+import * as fs from 'fs';
+
+const ensureWidgetsAreUnique = (sections: Section[]) => {
```

Comment:
I am not sure why this validation is needed, the sections aren't coming from our side?